lib/graphiti/resource/persistence.rb

```diff
@@ -177,16 +191,18 @@ def around_callback_proc(callbacks, index, *args, &blk)
         if callbacks[index + 1]
           proc do
             r = nil
+            args = args[0..0] if method(method_name).arity == 1
```

Comment:
Would like to move this arity check + args transformation into a utility method so that it's not repeated so much (and is more descriptive of the purpose).

Reply:
OK - will do what I can